spec/unit/verifier_spec.rb

```diff
@@ -167,6 +167,22 @@
     expect(result.errors.first).to eq "Expected file name: awesomelib.2013-09-08.manifest"
   end
 
+  it "detects invalid topics" do
```

Comment:
This test only detects one invalid topic. It would be good to have two tests, one for one invalid topic, and another one for multiple invalid topics.